packages/react-integration/demo-app-ts/src/components/demos/NotificationDrawerDemo/NotificationDrawerGroupsDemo.tsx

```diff
@@ -88,7 +96,7 @@ export class GroupsNotificationDrawerDemo extends React.Component<
     ];
     return (
       <NotificationDrawer>
-        <NotificationDrawerHeader count={4}>
+        <NotificationDrawerHeader count={4} onClose={this.onDrawerClose}>
```

Comment:
Can you also add the other new `closeIcon` prop to the  demo app.

Reply:
Added closeIcon to the groups demo integration app.